### PR TITLE
[SCAL-92013]-css updates

### PR DESCRIPTION
**Pull request by harsh-ts**
TicketId: https://thoughtspot.atlassian.net/browse/SCAL-92013

### CODE REVIEW

File: docs/src/pages/index.tsx [hunk 1]
```diff
@@ -226,10 +226,11 @@ const IndexPage = ({ location }) => {
         }
         return '100%';
     };
+    const shouldShowRightNav = params[TS_PAGE_ID_PARAM] !== 'introduction';
```
Comment (kailash-choudhary-thoughtspot) [inline docs/src/pages/index.tsx:229]: can we declare a const or Enum for 'introduction'

File: docs/src/components/Document/index.tsx [hunk 2]
```diff
@@ -17,15 +18,20 @@ const Document = (props: {
...
+        >
             <div
                 id={props.docTitle}
                 dangerouslySetInnerHTML={{
                     __html: props.docContent,
                 }}
             />
             {props.isPublicSiteOpen && <Footer />}
-        </div>
+        </div >
```
Comment (kailash-choudhary-thoughtspot) [inline docs/src/components/Document/index.tsx:34]: nit: extra space

File: docs/src/components/Document/index.scss [hunk 3]
```diff
@@ -215,6 +216,10 @@
     }
 }
 
+hr {
+    border-color: var(--border-color);
+}
```
Comment (kailash-choudhary-thoughtspot) [inline docs/src/components/Document/index.scss:221]: can we use div with top-border

File: docs/src/pages/index.tsx (hunk 1 above)
Comment (harsh-ts) [reply docs/src/pages/index.tsx:229]: I will make changes for them

File: docs/src/components/Document/index.scss (hunk 3 above)
Comment (harsh-ts) [reply docs/src/components/Document/index.scss:221]: Gatsby plugin transpiles adoc to html and there we have used `hr` tag for giving line under the element. we are giving matching css to it here

Comment (harsh-ts) [reply docs/src/components/Document/index.scss:221]: @ShashiSubramanya will be changing all occurance of `tr`, and I have added .divider class to replace it. so we can merge this PR

Comment (ShashiSubramanya) [reply docs/src/components/Document/index.scss:221]: Tried the divider class and the bootstrap CSS updates. Both changes look good.

**Review by ShashiSubramanya — APPROVED** [review]
Looks good

**Pull request merged**